src/tokenserver/extractors.rs

```diff
@@ -163,13 +133,26 @@ mod tests {
 
     #[actix_rt::test]
     async fn test_invalid_tokenserver_request() {
-        let rsa = Rsa::generate(2048).unwrap();
-        let state = make_state(&rsa);
-        let bearer_token = "I am not a valid token";
+        let fxa_uid = "test123";
+        let verifier = {
+            let start = SystemTime::now();
+            let current_time = start.duration_since(UNIX_EPOCH).unwrap();
+            let token_data = TokenData {
+                user: fxa_uid.to_owned(),
+                client_id: "client id".to_owned(),
+                scope: vec!["scope".to_owned()],
+                generation: current_time.as_secs() as i64,
+                profile_changed_at: current_time.as_secs() as i64,
+            };
+            let valid = false;
+
+            MockOAuthVerifier { valid, token_data }
```

Comment:
We need to stub out the verifier, since we can't forge the JWT that comes from FxA